angular/core/services/user_help_service.coffee

```diff
@@ -18,3 +18,15 @@ angular.module('loomioApp').factory 'UserHelpService', ($sce, CurrentUser) ->
 
     helpVideoUrl: ->
       $sce.trustAsResourceUrl(@helpVideo())
+
+    tenTipsArticleLink: ->
+      switch CurrentUser.locale
```

Comment:
Not for this PR, but I'm seeing a pattern of hard coding urls in the client here, and think this wants to get pulled out into the AppConfig if it gets any more complex 💛